pkg/services/ngalert/notifier/channels/pushover.go

```diff
@@ -118,11 +97,24 @@ func newPushoverNotifier(fc FactoryConfig) (*PushoverNotifier, error) {
 			Settings:              fc.Config.Settings,
 			SecureSettings:        fc.Config.SecureSettings,
 		}),
-		tmpl:     fc.Template,
-		log:      log.New("alerting.notifier.pushover"),
-		images:   fc.ImageStore,
-		ns:       fc.NotificationService,
-		settings: settings,
+		tmpl:   fc.Template,
+		log:    log.New("alerting.notifier.pushover"),
+		images: fc.ImageStore,
+		ns:     fc.NotificationService,
+		settings: pushoverSettings{
+			userKey:          uk,
+			apiToken:         at,
+			alertingPriority: ap,
+			okPriority:       okp,
+			retry:            r,
+			expire:           e,
```

Comment:
I really prefer the more verbose approach of having the full variable name instead of one or two letters in this case.

Reply:
done 👍🏽